java/com/google/javascript/jscomp/JsCompiler.java

```diff
@@ -141,6 +142,9 @@ private int run(Iterable<String> args) throws IOException {
         case "--disable_property_renaming":
           disablePropertyRenaming = true;
           continue;
+        case "--experimental_dev_build":
```

Comment:
Yeah.. the naming of this option is pretty tough. I think this is good. The only other option I can think of is "--disable_harmless_optimizations" , since the goal is to disable as much as possible without missing frequent errors due to the optimizations. Anyway, up to you.

Reply:
I'm going to keep this name for now.